app.js

```diff
@@ -39,7 +39,7 @@ app.function('sample_function', async ({ client, inputs, fail }) => {
     });
   } catch (error) {
     console.error(error);
-    fail({ error: `Failed to handle a function request: ${error}` });
+    fail({ error: `Failed to handle a step request: ${error}` });
```

Comment:
I'm on the fence about this: the event itself is `function_executed`, so it's function-related (not UI/step related).

Reply:
How about this? But its basically avoiding the ambiguous terminology  

```suggestion
    fail({ error: `Failed sample step: ${error}` });
```

Reply:
```suggestion
    fail({ error: `Failed to complete the step: ${error}` });
```

I'm also on the fence but leaning towards "step" since this is the message sent via @USLACKBOT

Reply:
@zimeg I like your suggestion 💯